src/DIRAC/Core/Security/Locations.py

```diff
@@ -12,6 +12,23 @@
 g_SecurityConfPath = "/DIRAC/Security"
 
 
+def getTokenLocation():
+  """ Get the path of the currently active access token file
+  """
+  envVar = 'DIRAC_TOKEN_FILE'
+  if envVar in os.environ:
+    tokenPath = os.path.realpath(os.environ[envVar])
+    if os.path.isfile(tokenPath):
+      return tokenPath
+  # /tmp/JWTup_u<uid>
+  tokenName = "JWTup_u%d" % os.getuid()
+  if os.path.isfile("/tmp/%s" % tokenName):
```

Comment:
Have you looked if there are any standards around how to store these files? If not I would be tempted to consider:
1. `tempfile.gettempdir()`
2. Somewhere in `$HOME` (I don't think it's a good idea, but we should at least consider it)

Reply:
I did not find a generally accepted standard. There are different solutions.

Reply:
I'd go with the tmp

Reply:
Surely `tempfile.gettempdir()` instead of `/tmp`?

Reply:
One thing to worry about with `/tmp` is that both POSIX and Linux state:
> Applications shall be allowed to create files in this directory, but shall not assume that such files are preserved between invocations of the application.

This does happen in the context of containers so it's not just a hypothetical problem.

We should at least ask the people behind WLCG's JWT spec to see if they have a proposal as I guess XRootD is going to need to find these tokens some how? Or is the idea to always embed them in the URL (which will be a pain for users)?

Reply:
Information about the location of the token can be stored in a variable environment, for example:
`export BAERER_TOKEN=..` and `export TOKEN_FILE_PATH=..` added to bashrc.

Reply:
I used the variables from the [WLCG Bearer Token Discovery](https://zenodo.org/record/3937438) description